src/components/badge/NumberBadge.js

```diff
@@ -54,10 +50,13 @@ const Badge = ({
   value,
   ...props
 }) => {
-  const animation = useSpringTransition(
-    delayUtil(bin(isVisible), delay),
-    BadgeSpringConfig
-  );
+  const [delayedIsVisible, setDelayedIsVisible] = React.useState(isVisible);
```

Comment:
Now useSpringTransition accepts numbers or bools only thus I moved a delay to js